### ISSUE TITLE
Protocol Buffers 2.6.0

### ISSUE THREAD

**Issue by ghost** [chobie/php-protocolbuffers#50]
Ad support for Protocol Buffers 2.6.0 if not already functional.

**Comment by dcelasun** [comment]
No, 2.6.0 features (oneof etc.) are not supported. I'm not sure @chobie is even maintaining this anymore.

FWIW, the changelog for 2.6.0 is [here](https://groups.google.com/forum/#!topic/protobuf/lvI8-sWZbUY).

**Comment by ghost** [comment]
That would be a great loss. Hoping for the best.

**Comment by cypres** [comment]
Google might pick up the ball themselves for proto3, where they will be adding php support. https://groups.google.com/forum/#!topic/protobuf/ZRpcfmeGK6s

No telling how long that might take.

Protocolbuffers 2.6.0 support would be great.

**Comment by ghost** [comment]
That would be great, as we have a lot of PHP code communicating with Java (Micro)services in the backend.

**Comment by chobie** [comment]
yea, I'll plan to support 2.6 feature foreseeable feature. I'd like to use that. 
let me adjust my schedule :)

**Comment by dcelasun** [comment]
Woah, that's great news @chobie! I thought this was abandoned.

**Comment by ghost** [comment]
Second that, i was fearing i had too rebuild my Protocol Buffers Maven Project which covers Linux AND Windows execution paths. Lucky me :)

**Comment by chobie** [comment]
@dcelasun we uses this ext in production heavily. so I don't through away.
(but my life style has been slightly changed. I'm still seek my life and OSS balance)

**Comment by dcelasun** [comment]
I didn't mean anything bad by that :) We use this in production heavily as well.

And of course, work life balance is very important :)

**Comment by ghost** [comment]
Please do find your balance. You saved my work life balance with your project already :)
